### DISCUSSION TITLE
`graphql-codegen` doesn't run for un-named queries

### DISCUSSION THREAD

**Discussion by spences10** [jycouet/kitql#106] [Q&A] [answered]
@notrab noticed that the stores and types are not generated with an un-named query.

Is this by design or a bug?

**Comment by jycouet** [comment, marked as answer]
It's by design.
What will be the name of your store if you don't have a named operation?

**Reply by notrab** [reply]
@jycouet maybe it's a question for codegen, but is there a way to hook into codegen to provide some terminal feedback that nothing was generated for file X because it had an unnamed query/mutation?

**Reply by jycouet** [reply]
Yes, it's more a codegen topic. And yes it would be nice to have info in the terminal 👌
